help/develop/domains/chem/cheminformatics.md

```diff
@@ -109,38 +86,34 @@ columns ([code sample](https://public.datagrok.ai/js/samples/domains/chem/simila
-.
+input `Column` ([code sample](https://public.datagrok.ai/js/samples/domains/chem/similarity-scoring-scores)).
 
-Similarly to the substructure-library based method of `searchSubstructure`, these functions maintain a cache of
+Similarly to the `searchSubstructure` function, these functions maintain a cache of
 pre-computed fingerprints. Once the function `findSimilar` or `getSimilarities` meets the `column` previously unmet, it
-builds the cache of computed fingerprints for the molecules of this `column`, and later uses it to compute similarity
+builds the cache of computed fingerprints for the molecules in `column`. Later it is used to compute similarity
 scores as long as the function is invoked for the same `column`.
 
 To only build (aka prime) this cache without performing an actual search, e.g. in case of pre-computing the fingerprints
-in the UI, the function should be called passing `molecule` either as `null` or as an empty string `''`.
+in the UI, call the function passing `molecule` as an empty string `''`.
 
-Sometimes it happens that the molecule strings aren't supported by RDKit-JS. These inputs are skipped when indexing, but
-an error log entry is added for each such entry to the browser's console together with the actual molecule string. For
-such unsupported molecule, the corresponding fingerprint of all zeroes is produced. Thus, the output Column (or
-DataFrame) is *always* of the input `column`'s length (with the number of rows equal to `column`'s length, respectively)
+In case molecule strings are not supported by RDKit JS they are skipped when indexing. For
+such unsupported molecules, null is returned instead of a fingerprint. Thus, the output Column (or
+DataFrame) is *always* of the input `column`'s length.
 .
 
 Examples (see on [GitHub](https://github.com/datagrok-ai/public/tree/master/packages/ApiSamples/scripts/domains/chem))
 
 * [Calculating descriptors](https://public.datagrok.ai/js/samples/domains/chem/descriptors)
 * [Substructure search](https://public.datagrok.ai/js/samples/domains/chem/substructure-search)
-* [Similarity search](https://public.datagrok.ai/js/samples/domains/chem/similarity-search)
 * [Diversity search](https://public.datagrok.ai/js/samples/domains/chem/diversity-search)
-* [Maximum common substructure](https://public.datagrok.ai/js/samples/domains/chem/mcs)
+* [Most common substructure](https://public.datagrok.ai/js/samples/domains/chem/mcs)
 * [R-Group analysis](https://public.datagrok.ai/js/samples/domains/chem/r-group)
 
 ### Molecule rendering
 
 #### SVG rendering with OpenChemLib
 
-With `grok.chem.svgMol` it's possible to render a molecule into a `div` container with an SVG render in it,
-using [OpenChemLib](https://github.com/cheminfo/openchemlib-js) library:
+Use `grok.chem.svgMol` to render a molecule into a `div` as an SVG, using [OpenChemLib](https://github.com/cheminfo/openchemlib-js) library:
```

Comment:
Puprose should be first
`To render a molecule, use grok.chem.svgMol.`

Reply:
Done